ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -985,13 +998,45 @@ Rectangle {
                 qsTr("Image files (%1)").arg(UrlUtils.validImageNameFilters)
             ]
             onAccepted: {
-                imageBtn.highlighted = false
                 validateImagesAndShowImageArea(fileUrls)
                 messageInputField.forceActiveFocus()
+                destroy()
             }
-            onRejected: {
-                imageBtn.highlighted = false
+            onRejected: destroy()
+            Component.onDestruction: d.imageDialog = null
+        }
+    }
+
+    Component {
+        id: chatCommandMenuComponent
+
+        StatusMenu {
+            id: chatCommandMenu
+            StatusAction {
+                text: qsTr("Add image")
+                icon.name: "image"
+                onTriggered: control.openImageDialog()
+            }
+
+            MouseArea {
+                implicitWidth: paymentRequestMenuItem.width
+                implicitHeight: paymentRequestMenuItem.height
+                hoverEnabled: true
+                visible: control.paymentRequestEnabled
```

Comment:
The naming is a bit confusing here. With `paymentRequestEnabled` set to true we can have payment actually disabled.

Maybe pair like:
```
bool paymentRequestsVisible
bool paymentRequestsDisabledInTestMode
```
(probably still not ideal but maybe bit better)